src/cli-constant.js

```diff
@@ -99,6 +100,8 @@ Available options:
                            Print trailing commas wherever possible when multi-line. Defaults to none.
   --parser <flow|babylon|typescript|postcss|json|graphql>
                            Specify which parse to use. Defaults to babylon.
+  --require-pragma <none|format|prettier>
```

Comment:
I would just name it `--pragma`, it feels weird to have the `require-` in front.

Reply:
Given the fact that we want it to be a boolean, disregard my renaming suggestion.